### PR TITLE
Add an accessibility audit procedure

### PR DESCRIPTION
**Pull request by sandbergja**

### CODE REVIEW

File: Accessibility/audit_procedure.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Auditing a page or site for accessibility
+
+1. Define the scope of the audit.
+2. On Google Drive, [create a copy of the Accessibility audit detailed report template](https://docs.google.com/spreadsheets/d/1xTNkQI-rHp0LzyjtBDRGtHFNdyG454BihGmjl71N2bA/copy).
```
Comment (maxkadel) [inline Accessibility/audit_procedure.md:4]: When I tried to go to this link, I got an "access denied" message for mk8066@princeton.edu (the address associated with my Princeton google account) - might need to check permissions.

**Review by maxkadel — CHANGES_REQUESTED** [review]
One link needs a permissions check, otherwise looks good, thanks @sandbergja! 🍄

File: Accessibility/audit_procedure.md (hunk 1 above)
Comment (sandbergja) [reply Accessibility/audit_procedure.md:4]: Good catch!  Fixed.

**Review by maxkadel — APPROVED** [review]
Thanks @sandbergja!

**Pull request merged**